MatrixGame/src/MatrixGame.h

```diff
@@ -5,16 +5,12 @@
 
 #pragma once
 
-#include "Resource.h"
-#include <shlobj.h>
-//#include "windows.h"
+#include "MatrixFormGame.hpp"
+#include "MatrixGameDll.hpp"
```

Comment:
i believe this is not supposed to be here.
`SRobotsSettings` struct predeclaration should be enough for current implementation.